### ISSUE TITLE
Corrupt colorize.lua file.

### ISSUE THREAD

**Issue by zenlord** [Kiwi/clyde#117]
## After upgrading pacman to 3.5 (and rebuilding the pacman db), I get the following error with clyde:

```
$ clyde
lua: error loading module 'clydelib.colorize' from file '/usr/share/lua/5.1/clydelib/colorize.lua':
    /usr/share/lua/5.1/clydelib/colorize.lua:1: unexpected symbol
stack traceback:
    [C]: ?
    [C]: in function 'require'
    /usr/bin/clyde:4: in main chunk
    [C]: ?
```

Rebuilding clyde from git did not change anything.

**Comment by juster** [comment]
Actually this seems to be an error completely unrelated to pacman 3.5. Your colorize file is empty I presume?

(edited your post ttitle and indented your console example to look nicer)

PS we're working on getting clyde to work with pacman 3.5

**Comment by zenlord** [comment]
I guess I was busted when cutting corners: I had encountered the same problem as issue 115, but I was not on my Arch-machine when submitting the bug. When I came across a post with (at first glance) the same error, I just copy/pasted it from that forum post.

My issue is a duplicate of issue 115, albeit that my pacman-version is from core, and not from testing. THX for the head's up (what was I thinking: trying to cheat the writer of the program... _Duh_)
